using_images/other_images/jenkins.adoc

```diff
@@ -435,6 +435,18 @@ control the label field of the Kubernetes plug-in pod template by setting an
 annotation on the image stream or image stream tag object with the key
 `slave-label`. Otherwise, the name is used as the label.
 
+[NOTE]
+====
+Keep in mind that after the Pod Template is created, if you log into the Jenkins
```

Comment:
s/Keep in mind that/  (redundant since it's already going to be marked as a note)